pkg/operator/ceph/csi/spec.go

```diff
@@ -105,7 +106,7 @@ var (
 // manually challenging.
 var (
 	// image names
-	DefaultCSIPluginImage   = "quay.io/cephcsi/cephcsi:v3.2.1"
+	DefaultCSIPluginImage   = "quay.io/cephcsi/cephcsi:canary"
```

Comment:
Can't we just wait for the upcoming 3.3 release which should go with Rook 1.6?

Reply:
> Can't we just wait for the upcoming 3.3 release which should go with Rook 1.6?

@leseb  Those changes were just to check everything worked with host network disabled. I will change them back after cephcsi 3.3 release.